kevm-pyk/src/kevm_pyk/kdist/plugin.py

```diff
@@ -79,6 +85,7 @@ def _build_env(self) -> Iterator[Path]:
             'syntax_module': 'ETHEREUM-SIMULATION',
             'optimization': 2,
         },
+        deps=('plugin',),
```

Comment:
I believe that haskell target may also require `plugin`? If the booster is enabled?

Reply:
This entry defines kompilation of `ETHEREUM-SIMULATION` via  `kevm_kompile(target=KompileTarget.HASKELL, ...)`, for which `plugin` is not needed. It is indeed necessary for `kevm_kompile(target=KompileTarget.HASKELL_BOOSTER, ...)` though.